src/js/components/Ballot/BallotElectionList.jsx

```diff
@@ -6,31 +6,31 @@ let moment = require("moment");
 export default class BallotElectionList extends Component {
 
   static propTypes = {
-    ballot_election_list: PropTypes.array.isRequired,
-    _toggleSelectBallotModal: PropTypes.func.isRequired,
-  }
+    ballotElectionList: PropTypes.array.isRequired,
+    toggleFunction: PropTypes.func.isRequired,
+  };
 
   constructor (props){
     super(props);
     this.state = {};
   }
 
-  updateBallot (original_text_for_map_search, simple_save, google_civic_election_id) {
-    console.log("BallotElectionList.jsx updateBallot, google_civic_election_id: ", google_civic_election_id);
-    VoterActions.voterAddressSave(original_text_for_map_search, simple_save, google_civic_election_id);
-    // Not necessary here: BallotActions.voterBallotItemsRetrieve(google_civic_election_id);
-    this.props._toggleSelectBallotModal();
+  updateBallot (originalTextForMapSearch, simpleSave, googleCivicElectionId) {
+    console.log("BallotElectionList.jsx updateBallot, googleCivicElectionId: ", googleCivicElectionId);
```

Comment:
Hi @SailingSteve, I will go ahead and approve this pull request, but could you refactor these variable names? I'd prefer that these internal variables remain all lower case with "_" between words instead of lower camel case. I'll update the coding standards documentation with this:
## Variable Naming Standards
* Please name variables all lower case with "_" in between words.
* Please use full words instead of abbreviations (Ex/ Please use “person” instead of “per” or “p”.)
* Exception: Please use lower case camel case ("camelCaseLettering") for local variables passed into components as properties, unless you are passing a widely used data variable (like "google_civic_election_id"), especially if it is passed around to multiple components.